tests/datalab/datalab/end_to_end_tests/test_all_identical_examples.py

```diff
@@ -64,25 +123,216 @@ def test_issue_detection(self, dataset):
             atol=5e-16,
         )
 
+        label_issues = lab.get_issues("label")[["is_label_issue", "label_score"]]
+        expected_label_issues = pd.DataFrame(
+            [{"is_label_issue": False, "label_score": 1.0}] * (N - 1)
+            + [
+                {"is_label_issue": False, "label_score": 0.0}
```

Comment:
Note that this point is not flagged as having a label issue, but the LQS is at a minimum.
It's the only point that disagrees with other points.

@jwmueller I suppose that is the expected behavior for a dataset:

```python
labels = ["Class A"] * 99 + ["Class B"]
pred_probs = np.array([[1.0, 0.0] * 100])  # All points predicted to be "Class A"
```